CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 A breaking change should be clearly marked in this log.
 
+#### 0.6.2
+* replaced `getLedgerEntry` with `getLedgerEntries`. `getLedgerEntry` is deprecated and should no longer be used.
+
```

Comment:
lol sorry, this isn't ready for release / a final decision on version bump. you should keep it in the "Unreleased" section and we'll tag it when we're ready. also marking it as breaking more explicitly!

```suggestion
A breaking change should be clearly marked in this log.

* **Breaking Change**: Replaced `getLedgerEntry` with `getLedgerEntries`. `getLedgerEntry` is deprecated and should no longer be used ([#66](https://github.com/stellar/js-soroban-client/pull/66)).

```